firefox-ios/Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,14 +1,5 @@
 {
   "pins" : [
-    {
```

Comment:
@razvanlitianu do we need this? seems like this got pushed accidentally 🤔

Reply:
Correct, SwiftPM is acting up lately. Will revert